internal/net/network_service.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: 2023 Comcast Cable Communications Management, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package net
+
+import (
+	"go.uber.org/fx"
+	"net"
+)
+
+type NetworkServiceIn struct {
+	fx.In
+}
+
+type NetworkServiceOut struct {
+	fx.Out
+	NetworkService *NetworkService
+}
+
+var NetworkServiceModule = fx.Module("networkService",
+	fx.Provide(
+		func(in NetworkServiceIn) *NetworkService {
+			return &NetworkService{
+				n: NewNetworkWrapper(),
+			}
+		}),
+)
```

Comment:
this repo started keeping the `fx` related code in the `cmd/xmidt` as their files.

I don't have a preference between the current or this pr's pattern, but I'm assuming we want to keep things consistent.

Reply:
good point. I'll switch it.  we are def not consistent between repos